README.md

```diff
@@ -76,6 +76,7 @@ public final class UserService {
     // MARK: Initialization
 
     /// Public, memberwise initializer that takes each injected property.
+    /// This initializer can be auto-generated by SafeDI if it isn‘t manually written.
```

Comment:
Seemed like a good thing to add, though it is unrelated to the main thrust of this PR.

Reply:
Shouldn't it be this? "This initializer **will** be auto-generated..."

Reply:
It is only auto-generated if certain conditions are met. Specifically, we generate the initializer if there is no uninitialized, non-optional SafeDI property on the type. Otherwise we can't generate an initializer that will compile.